src/main/java/org/opensearch/ad/common/exception/NotSerializedADExceptionName.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.ad.common.exception;
+
+import org.opensearch.OpenSearchException;
+
+/**
+ * OpenSearch restricts the kind of exceptions that can be thrown over the wire
```

Comment:
Also, does over the wire refer to return exception to client or return exception across transport layer within cluster?

Reply:
over the wire means transferred using network.  So the exception comes from a remote node.